src/vs/editor/common/languages/defaultDocumentColorsComputer.ts

```diff
@@ -101,7 +101,7 @@ function _findMatches(model: IDocumentColorComputerTarget | string, regex: RegEx
 function computeColors(model: IDocumentColorComputerTarget): IColorInformation[] {
 	const result: IColorInformation[] = [];
 	// Early validation for RGB and HSL
-	const initialValidationRegex = /\b(rgb|rgba|hsl|hsla)(\([0-9\s,.\%]*\))|\s+(#)([A-Fa-f0-9]{6})\b|\s+(#)([A-Fa-f0-9]{8})\b|^(#)([A-Fa-f0-9]{6})\b|^(#)([A-Fa-f0-9]{8})\b/gm;
+	const initialValidationRegex = /\b(rgb|rgba|hsl|hsla)(\([0-9\s,.\%]*\))|\s+(#)([A-Fa-f0-9]{6})\b|\s+(#)([A-Fa-f0-9]{8})\b|^(#)([A-Fa-f0-9]{6})\b|^(#)([A-Fa-f0-9]{8})\b|(?<=['":=\[({])(#)([A-Fa-f0-9]{6})\b|(?<=['":=\[({])(#)([A-Fa-f0-9]{8})\b/gm;
```

Comment:
Hi thanks for the change. I want the hex regex to be changed so that is simply adds the case when the length 6 or 8 color is inside of a string (or after a single or double quote). You don't need to cover the case where it follows other delimiters.